### ISSUE TITLE
Mention all users in a voice channel

### ISSUE THREAD

**Issue by karimawi** [Vencord/plugin-requests#253]
### Plugin Description

Sometimes when I'm in a VC with friends, I want to send a photo or a video for them to look at, some of them might be deafened, some might have dnd or their notifications set to only mention, I would have to say "look in general" or "check discord" for them to see what I sent, or maybe you're not even in the same voice but want to mention those users specifically.

**_Plugin Idea:_**
Mention all users (excluding self) currently in voice channel by mentioning the channel, there are several approaches like a custom slash command with an input for the VC channel which I think would be thee best option, channel context menu "mention all users" to put it in the currently visible text channel for me to then add to the message, or if it'd be easier "copy all users mention" to just paste it before putting your message

**_Logic (Pseudocode-ish):_**
- slash command executes with voice channel entity input
- get current users in the input voice channel
- concatenate mentions and add them to the message box (similar to QuickMention)

**_Screenshots:_**
![image](https://github.com/Vencord/plugin-requests/assets/69881381/07b6db98-3feb-4b8e-84c5-257eb2c7f952)

![image](https://github.com/Vencord/plugin-requests/assets/69881381/02e4513a-48a6-45aa-939f-1472f5171ecf)
↓
![image](https://github.com/Vencord/plugin-requests/assets/69881381/e2527329-0879-42c2-9f89-89c001d88078)

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Sqaaakoi** [comment]
I have considered making this plugin before. Considering how #! works to mention a voice channel in chat, would #@ be a good way to trigger it?

**Comment by karimawi** [comment]
yeah that's another way too apart from the slash command which I think would be easier

**Comment by EdVraz** [comment]
https://github.com/Vendicated/Vencord/pull/2341
